test/functional/wallet_multiwallet.py

```diff
@@ -173,17 +176,34 @@ def wallet_file(name):
 
         w1, w2, w3, w4, *_ = wallets
         node.generatetoaddress(nblocks=101, address=w1.getnewaddress())
-        assert_equal(w1.getbalance(), 100)
-        assert_equal(w2.getbalance(), 0)
-        assert_equal(w3.getbalance(), 0)
+        wallet_name = sorted(wallet_names)[1]
```

Comment:
Super nit (please feel free to ignore): Would it make sense to define a variable like `extern_wallet` next to where `wallet_names` is defined, instead of relying on the sort order of the path names?

Reply:
Not a bad idea; will look at this if I have to retouch the PR.

Reply:
Done in the latest rebase; it's much better with your suggestion.

Mind re-ACKing here and/or #18594 (the client-side version)?